docs/tutorials/autogluon-cloud.md

```diff
@@ -1,2 +1,119 @@
+**Make sure you review those files and make necessary changes according to your use case before applying them.**
+
+## Training
+Using `autogluon.cloud` to train AutoGluon backed models are simple and not too much different from training an AutoGluon predictor directly.
+
+Currently, `autogluon.cloud` supports training/deploying `tabular`, `multimodal`, `text`, and `image` predictors. In the example below, we use `TabularCloudPredictor` for demonstration. You can substitute it with other `CloudPredictors` easily as they share the same APIs.
+
+```{.python}
+from autogluon.cloud import TabularCloudPredictor
+train_data = 'train.csv'
+predictor_init_args = {label='label'}  # init args you would pass to AG TabularPredictor
+predictor_fit_args = {train_data, time_limit=120}  # fit args you would pass to AG TabularPredictor
+cloud_predictor = TabularCloudPredictor(
+    cloud_output_path='YOUR_S3_BUCKET_PATH'
+).fit(
+    predictor_init_args,
+    predictor_fit_args,
+    instance_type="INSTANCE_TYPE"  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+)
+```
+
+To retrieve general info about the fitted `CloudPredictor`
+
+```{.python}
+cloud_predictor.info()
+```
+
+### Reattach to a Previous Training Job
+If your local connection to the training job died for some reason, i.e. lost internet connection, your training job will still be going on SageMaker, and you can reattach to the job with another `CloudPredictor` as long as you have the job name.
+
+```{.python}
+another_cloud_predictor = TabularCloudPredictor(cloud_output_path='YOUR_S3_BUCKET_PATH')
+another_cloud_predictor.attach_job(job_name="JOB_NAME")
+```
+
+To be noticed, the reattached job will no longer give live stream of the training job's log. Instead, the log will be available once the training job is finished.
+
+## Endpoint Deployment and Real-time Prediction
+If you want to deploy a predictor as a SageMaker endpoint, which can be used to do real-time inference later, it is just one line of code:
+
+```{.python}
+cloud_predictor.deploy(
+    instance_type="INSTANCE_TYPE"  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+)
+```
+
+You can also attach to a deployed endpoint,
+
+```{.python}
+cloud_predictor.attach_endpoint(endpoint="ENDPOINT_NAME")
+```
+
+To do real-time prediction,
+
+```{.python}
+result = cloud_predictor.predict_real_time(
+    'test.csv',
+    instance_type="INSTANCE_TYPE"  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+)  # result will be a pandas dataframe
+```
+
+## Batch Inference
+When minimizing latency isn't a concern, then the batch inference functionality may be easier, more scalable, and more appropriate.
+To do a batch inference,
+
+```{.python}
+cloud_predictor.predict('test.csv')  # results will be stored in s3 bucket
+cloud_predictor.download_predict_results()  # download the results to your local machine
```

Comment:
Why do I need to download results this time? Did I need to download results for predict_real_time? What is `result`?

Reply:
So sagemaker endpoint will return results as payload because the results are small. Batch inference on the other hand store the results into s3 bucket. We just provide a util for the user to be able to download it. We cannot modify behavior of sagemaker

Reply:
should communicate to user in tutorial. Also, download to where?

Ideally should have an option akin to `download=True` so it returns `result` (suggestion)

Reply:
I'll add explanations. User can specify a `save_path`. If not specified, we will create a folder then.

I think your suggestion is good. Will work on a PR to add this functionality later.